pkgs/swiftgen/swift2objc/lib/src/temp_test/main.dart

```diff
@@ -0,0 +1,9 @@
+// swiftc source.swift -emit-module -emit-symbol-graph -emit-symbol-graph-dir .
+import '../parser/parser.dart';
+
+void main() {
+  final parser = Parser();
+  final bindings =
+      parser.parseSymbolgraph("lib/src/temp_test/symbolgraph/source.symbols.json");
+  print(bindings.declarations);
```

Comment:
I think it's fine to check in the symbols.json file (let's .gitignore the other outputs though).

Then you can convert this to a test by verifying the AST. Is the string that is printed here a complete representation? If so you could just hard code the expeced string.

Reply:
Um well actually this print line is there just so I can add a breakpoint to it and inspect the result in the debugger.

Reply:
Ok, well I guess we can wait until we're generating code, then verify the generated code for a hard coded symbol graph file. I'll add some github CI infra to autorun the tests once we have them.

Btw, I already added some basic checks. if you take a look at the checks section, you can see that the `swift2objc / analyze` check is failing. Looks like some files need some auto-formatting.

Reply:
done